.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 README.md
 out
+java-.iml
```

Comment:
you can ignore all .iml files instead of only java-

Reply:
done